ddtrace/_trace/utils_botocore/span_pointers/dynamodb.py

```diff
@@ -11,6 +11,7 @@
 from ddtrace._trace._span_pointer import _SpanPointerDescription
 from ddtrace._trace._span_pointer import _SpanPointerDirection
 from ddtrace._trace._span_pointer import _standard_hashing_function
+from ddtrace._trace.utils_botocore.span_pointers.telemetry import record_span_pointer_calcuation_issue
```

Comment:
since some of these warnings are actually due to missing table mapping configuration, those should probably still be warnings since they're very much in our users' hands. i'll update that once we've settled on an approach for the telemetry.

Reply:
this is done now.